sktime/transformations/series/impute.py

```diff
@@ -362,7 +362,7 @@ def _impute_with_forecaster(self, X, y):
                 )
 
                 # replace missing values with predicted values
-                X[col][na_index] = self._forecaster.predict(fh=fh, X=y)
+                X.loc[col, na_index] = self._forecaster.predict(fh=fh, X=y)
```

Comment:
Not sure if I'm missing something, but shouldn't it be like this?

```suggestion
                X.loc[na_index, col] = self._forecaster.predict(fh=fh, X=y)
```

Nothing is failing in CI so far, but I am not following the existing change.